calculator/operations.py

```diff
@@ -1,18 +1,21 @@
-
+from __future__ import division
+from functools import *
 def add(*args):
-    # your implementation here
-    pass
+    #print("my args is {}".format(args))
+    result = reduce(lambda acc, y: acc + y, args, 0)
+    return result
 
 def subtract(*args):
     # your implementation here
-    pass
-
+    result = reduce(lambda acc, y: acc - y, args[1:], args[0])
+    return result
+    
 def multiply(*args):
-    # your implementation here
-    pass
+    result = reduce(lambda acc, y: acc * y, args, 1)
+    return result
 
 def divide(*args):
-    # your implementation here
-    pass
+     result = reduce(lambda acc, y: acc / y, args[1:], args[0])
+     return result
 
 # add your custom operations here
```

Comment:
Nice use of functional programming in here with `reduce()`!  The consistency makes it really readable.